packages/validator/src/cli-validator/run-validator.js

```diff
@@ -234,12 +238,40 @@ async function runValidator(cliArgs, parseOptions = {}) {
       continue;
     }
 
-    // Compute scoring information if the user requested the "impact score" option,
-    // or if JSON output is requested. The JSON output always includes all results,
-    // both the standard rule violations and the scoring information.
+    // Compute scoring information if 1) the user requested the "impact score"
+    // option, 2) if JSON output is requested, or 3) if the markdown report is
+    // requested. The JSON output and markdown report always include all results,
+    // including the standard rule violations and the scoring information.
     let impactScoreInformation = {};
-    if (context.config.produceImpactScore || outputIsJSON(context)) {
+    if (
+      context.config.produceImpactScore ||
+      outputIsJSON(context) ||
+      context.config.markdownReport
+    ) {
+      logger.info('Impact scores are being calculated...');
       impactScoreInformation = await produceImpactScore(results, context);
+    } else {
+      logger.info(
+        'Impact scores are not being calculated. Scores are calculated when' +
+          'requested, or when JSON output or a Markdown report is requested.'
+      );
+    }
+
+    // Combine validator and impact score results into one object.
+    results = {
+      ...results,
+      impactScore: {
+        ...impactScoreInformation,
+      },
+    };
+
+    // If the user requested a markdown report of the results, print that here.
+    // Note that we need to do this before we check if the "summaryOnly" option
+    // was provided, because that will filter the results and we want to print
+    // all of the results in the report.
+    let markdownReportLocation = false;
```

Comment:
Beware, picky comment: I know JS is loosely typed and this code works fine, but it might be better to use a value that's "closer" to strings, since the name of the variable also implies that it will hold the a path to the file. With my somewhat limited JS knowledge, I would consider using `undefined`, `null` or maybe and empty string - instead of a boolean.

Reply:
Fixed!